hazelcast-client/src/main/java/com/hazelcast/client/config/ClientIcmpPingConfig.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2008-2017, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.client.config;
+
+/**
+ * ICMP ping is used to detect if machine that a remote hazelcast member runs on alive or not
+ * ICMP Ping detector will attempt 3 `max-attempts` pings, one every 1000 millis `interval-milleseconds`
```

Comment:
Why javadoc has hardcoded properties, such as `3` or `1000` milis?

Reply:
Also do not use markdown formatting in javadoc. Use `<code>...</code>` or `{@code ...}` instead.

Reply:
Those were examples and they are also default values but I think they are already mentioned in related methods.